app/static/APISpecification.yaml

```diff
@@ -100,6 +102,235 @@ paths:
           $ref: '#/components/responses/404'
         '500':
           $ref: '#/components/responses/500'
+  /rapid:
+    get:
+      summary: Resolve rapid site url
+      description: Resolves to Ensembl site url (home page).
+      tags:
+        - Rapid Resolver
+      responses:
+        '301':
+          description: Redirect to resolved URL
+          headers:
+            Location:
+              schema:
+                type: string
+              description: Ensembl URL
+          content: {}
+        '200':
+          description: OK
+          headers:
```

Comment:
This response (header & content) is repeated in multiple places, good candidate for a component.